bundles/org.openhab.binding.tr064/README.md

```diff
@@ -127,4 +127,22 @@ If only a specific phonebook from the device should be used, this can be specifi
 The default is to use all available phonebooks from the specified thing.
 In case the format of the number in the phonebook and the format of the number from the channel are different (e.g. regarding country prefixes), the `matchCount` parameter can be used.
 The configured `matchCount` is counted from the right end and denotes the number of matching characters needed to consider this number as matching.
+A `matchCount` of `0` is considered as "match everything".
 
+## Rule Action
+
+The phonebooks of a `fritzbox` thing can be used to lookup a number from rules via a thing action:
+
+`String name = phonebookLookup(String number, String phonebook, int matchCount)`
+
+`phonebook` and `matchCount` are optional parameters.
```

Comment:
I just tested those changes and based on the description here would have expected that something like 
```
val tr064Actions = getActions("tr064","tr064:fritzbox:2a28aee1ee")
val result = tr064Actions.phonebookLookup("01234567")
``` 
would actually work as the other parameters are described as optional. 

But the above rule code results in an error like
```
Script execution of rule with UID 'fritzbox' failed: An error occurred during the script execution: index=3, size=3 in fritzbox
```

Using `val result = tr064Actions.phonebookLookup("01234567", null, null)` works fine. Maybe might be good to update the readme here, so it's clear the parameters can't be ommited, but need to be set to `null` instead - or update the code if the parameters should be ommitable.

Reply:
Probably adding those methods would be the better option. Will do that tomorrow. Is it working otherwise as expected?

Reply:
Only did some simple lookups of incoming call numbers in the phonebook. That worked as expected. But didn't test (yet) if the `phonebook` and `matchCount` parameters are working correctly.

Reply:
Just tested looking up with specific phonebooks. Seems to work as expected. But might be useful to mention it's selected by the phonebooks name (and not its id)